src/lib/components/layout/FormLayout/FormLayout.jsx

```diff
@@ -70,9 +65,8 @@ export const FormLayout = (props) => {
         styles.root,
         fieldLayoutClass(fieldLayout),
         fieldLayout === 'horizontal' ? labelWidthClass(labelWidth) : '',
-        labelWidthFallback ? styles.hasRootCustomLabelWidthFallback : '',
       ].join(' ')}
-      style={inlineStyle(customLabelWidth, labelWidthFallback)}
+      style={getInlineStyle(customLabelWidth)}
```

Comment:
I would try to get rid of the `customLabelWidth` var and the `getInlineStyle` function. Like (not tested):

```suggestion
    {...hasCustomLabelWidth ? { style: { '--rui-custom-label-width': labelWidth }} : {}}
```